wireless_modules/mqtt_test/main.py

```diff
@@ -0,0 +1,108 @@
+import time
+import machine
+import config
+import ubinascii
+import fake
+from machine import Pin
+
+# Use micro pip to install packages
+import upip
+upip.install("micropython-umqtt.simple2")
+from umqtt.simple import MQTTClient
+
+
+# Define module number and MQTT topics
+MODULE_NUM = "1"
+FAKE_MODULE = fake.module(MODULE_NUM)
+
+PUB_DATA = b'/v3/wireless-module/{}/data'.format(MODULE_NUM)
```

Comment:
Does whatever version of micropython we use support f-strings?

Reply:
No I dont believe so as I had trouble getting them to work.